src/main/java/com/mojang/serialization/JsonOps.java

```diff
@@ -44,7 +44,7 @@ public <U> U convertTo(final DynamicOps<U> outOps, final JsonElement input) {
         if (input instanceof JsonArray) {
             return convertList(outOps, input);
         }
-        if (input instanceof JsonNull) {
+        if (input instanceof JsonNull || input == null) {
```

Comment:
```suggestion
        if (input == null || input instanceof JsonNull) {
```
This takes advantage of Java’s short-circuit evaluation:
– if input is null, only the first condition `(input == null)` is evaluated
– if input is not null, both conditions are checked as needed

In contrast, with the original order:
```
if (input instanceof JsonNull || input == null)
```
the `instanceof` check is always evaluated first, even when input is `null`,
which is unnecessary (though completely safe).